h2o-py/tests/testdir_algos/gam/pyunit_PUBDEV_7367_random_gridsearch_invalid.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import division
```

Comment:
Karthik:  Please add one more invalid test with you adding gam_columns to both the hyper-parameters and the hyper_parameters['constraints'] to make sure it will trigger your new error conditions.